_posts/2020-04-03-what-does-appsync-do.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: What does AWS AppSync do ?
+date: 2020-04-03
+tags: AWS AppSync Cloudformation
+author: Shermayne Lee
+---
+
+# Overview
+
+AWS AppSync and GraphQL have gained popularity in recent years. However it can be confusing and difficult to start for developers new to graphQL concept.
+
```

Comment:
You probably need a quick description of what GraphQL is, and a link for more information